tests/unit-tests/platforms/x11/test_platform.cpp

```diff
@@ -69,6 +69,7 @@ class X11GraphicsPlatformTest : public ::testing::Test
     {
         return std::make_shared<mg::X::Platform>(
             std::make_shared<mtd::MockX11Resources>(),
+            std::string("Mir on X"),
```

Comment:
Since its introduction in C++11 we tend to use uniform initialization syntax (braces for initialization) everywhere.

But here there is no need as there is a converting constructor.

```suggestion
            "Mir on X",
```